bootstrap-salt.sh

```diff
@@ -785,6 +785,20 @@ if [ -d "${_VIRTUALENV_DIR}" ]; then
     exit 1
 fi
 
+# Set the python exe if unset and version is >= 2019.2
+if [ "$_PY_EXE" = "" ] && [ "$(echo "$STABLE_REV" | grep -E '^(latest|2019\.2|3000)$')" != "" ]; then
+    which python3 > /dev/null
```

Comment:
Use `command` instead of `which`. See source code for examples.

Reply:
There is a special function for that: `__check_command_exists`

Reply:
Even better :smile: